src/frontend/webpack.config.js

```diff
@@ -63,6 +64,7 @@ module.exports = (env) => {
     resolve: {
       // Add '.ts' and '.tsx' as resolvable extensions.
       extensions: ['.ts', '.tsx', '.js', '.json'],
+      plugins: [new TsconfigPathsPlugin({})],
       modules: [
         path.resolve(__dirname, 'js'),
         __dirname,
```

Comment:
WIth the use of `TsconfigPathsPlugin`, we could remove `js` and `__dirname` from modules array.
```suggestion
      modules: [
```